### DISCUSSION TITLE
Hello my ips are blocked

### DISCUSSION THREAD

**Discussion by sbkapase** [community/community#145098] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### GitHub Feature Area

Security and Privacy

### Body

Hello

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @sbkapase ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category and including more detail in your post like which GitHub feature(s) are affected. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
